src/coreComponents/physicsSolvers/fluidFlow/FlowSolverBase.cpp

```diff
@@ -249,6 +250,53 @@ void FlowSolverBase::initializePreSubGroups()
   }
 }
 
+void FlowSolverBase::validatePoreVolumes( DomainPartition const & domain ) const
+{
+  real64 minPoreVolume = 0.0;
```

Comment:
`minPoreVolume` needs to be initialized with a non-zero value as `poreVolumeThreshold = 1e-4`; if no negative pore volume in the mesh, it will always report that the minimum pore volume is 0.

Reply:
Thanks for the catch! This is fixed now